core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -269,13 +269,19 @@ trait AttributeComponent {
       }
     }
 
-    def findUniqueStringsByNameQuery(attrName: AttributeName, queryString: String) = {
-      filter(rec =>
+    def findUniqueStringsByNameQuery(attrName: AttributeName, queryString: Option[String]) = {
+      // ToDo: figure out what the generated query is
+
+      val basicFilter = filter(rec =>
         rec.namespace === attrName.namespace &&
-        rec.name === attrName.name &&
-        rec.valueString.isDefined &&
-        rec.valueString.like(s"%$queryString%")
-      ).map(_.valueString).distinct
+          rec.name === attrName.name &&
+          rec.valueString.isDefined)
+
+      (queryString match {
+        case Some(query) => basicFilter.filter(_.valueString.like(s"%${query}%"))
```

Comment:
case sensitivity is defined by the db collation, which is `utf8_general_ci`, at least in dev. The "ci" in the collation signifies case-insensitive. So if we ever change our db implementation or db collation, this could be different ... but I'm sure we'd have a ton of other repercussions as well.